js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -464,3 +464,12 @@ function removeFromMessageQueue(queueObj) {
     mabMessageQueue.splice(messageObjIndex, 1);
   }
 }
+
+function messageTrimmingBannerText(message: MessageToContact, maxLength = 128) {
+  const hintString = message.bannerMessage.trim();
```

Comment:
Since `bannerMessage` is optional, you need to handle that case to avoid an exception. Since we're using typescript, I think you could use optional chaining:
```suggestion
  const hintString = message.bannerMessage?.trim() ?? '';
```